backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -161,6 +161,7 @@ private UserInfo reprovisionUser(
     apiUser.setNameInfo(name);
     apiUser.setIsDeleted(false);
     apiUser.setFacilities(facilitiesFound);
```

Comment:
(non-blocking) should we create a private method that sets the facilities and the roles? 🤔 If you think that doesn't affect readability

Reply:
can you tell me more about what you mean by this? if the method is private how can we call it from the user service?

Reply:
OH are you suggesting a private method in ApiUserService that does both since we're always updating them simultaneously?

Reply:
Yes! That's what I was thinking but I think it's fine as-is... I might do something for that when working on #7598... we'll see 😅

Reply:
I think that would make sense! especially with whatever you add we may start to see a bigger block of duplicated code